### DISCUSSION TITLE
More CSS examples coming? Or will many components remain only Tailwind?

### DISCUSSION THREAD

**Discussion by rchrdnsh** [melt-ui/melt-ui#760] [General]
So, not a tailwind person myself, and in quite a few of the components there is no example CSS, only example tailwind...is this going to change at some point in the future?

Not really interested in tailwind ATM, personally.

Thanks! XD

**Comment by TGlide** [comment]
See https://github.com/melt-ui/melt-ui/issues/574 :)

**Reply by rchrdnsh** [reply]
Schweet, thank you :-)
